Sources/MockoloFramework/Templates/ParamTemplate.swift

```diff
@@ -20,23 +20,28 @@ extension ParamModel {
     func applyParamTemplate(name: String,
                             label: String,
                             type: SwiftType,
-                            inInit: Bool) -> String {
+                            inInit: Bool,
+                            isGeneric: Bool) -> String {
         var result = name
         if !label.isEmpty {
             result = "\(label) \(name)"
         }
-        if !type.isUnknown {
+        if isGeneric {
+            // FIXME: `type` is used as a generic constraint
```

Comment:
This is caused by wrong `ParamModel` architecture, but fixing this is outside the scope of this PR.